src/Microsoft.OpenApi/Models/OpenApiOperation.cs

```diff
@@ -248,17 +248,25 @@ public void SerializeAsV2(IOpenApiWriter writer)
                     if (consumes.Contains("application/x-www-form-urlencoded") ||
                         consumes.Contains("multipart/form-data"))
                     {
-                        parameters.AddRange(RequestBody.ConvertToFormDataParameters(writer));
+                        var formDataParameters = RequestBody.ConvertToFormDataParameters(writer);
+                        if (formDataParameters != null)
```

Comment:
```suggestion
                        if (RequestBody.ConvertToFormDataParameters(writer) is {} formDataParameters)
```
nit: you can do something like that, which not only reduces allocations but also allows you to combine the condition with the parent one